src/microjson/model.py

```diff
@@ -9,141 +9,155 @@
 
 class GeoAbstract(BaseModel):
     """Abstract base class for all GeoJSON objects"""
+
     bbox: Optional[List[float]] = Field(None, min_length=4)
 
 
 class Point(GeoAbstract):
     """A GeoJSON Point object"""
+
     type: Literal["Point"]
-    coordinates: Coordinates
+    coordinates: Coordinates  # type: ignore
 
 
 class MultiPoint(GeoAbstract):
     """A GeoJSON MultiPoint object"""
+
     type: Literal["MultiPoint"]
-    coordinates: List[Coordinates]
+    coordinates: List[Coordinates]  # type: ignore
 
 
 class LineString(GeoAbstract):
     """A GeoJSON LineString object"""
+
     type: Literal["LineString"]
-    coordinates: List[Coordinates]
+    coordinates: List[Coordinates]  # type: ignore
 
 
 class MultiLineString(GeoAbstract):
     """A GeoJSON MultiLineString object"""
+
     type: Literal["MultiLineString"]
-    coordinates: List[List[Coordinates]]
+    coordinates: List[List[Coordinates]]  # type: ignore
 
 
 class Polygon(GeoAbstract):
     """A GeoJSON Polygon object"""
+
     type: Literal["Polygon"]
-    coordinates: List[List[Coordinates]]
+    coordinates: List[List[Coordinates]]  # type: ignore
 
 
 class MultiPolygon(GeoAbstract):
     """A GeoJSON MultiPolygon object"""
+
     type: Literal["MultiPolygon"]
-    coordinates: List[List[List[Coordinates]]]
+    coordinates: List[List[List[Coordinates]]]  # type: ignore
```

Comment:
What is the main reason for ignoring mypy check???

Reply:
Mypy will complain on the type. It will say that they are incompatible types